app/dashboard/settings/ui/settings-cards.tsx

```
@@ -0,0 +1,81 @@
+'use client';
+
+import { Card, CardContent, CardDescription, CardHeader, CardTitle } from '@/components/ui/card';
+import { UserSettingsForm } from '../user-settings-form';
+import { ClerkManageAccountButton } from '@/components/auth/clerk-manage-account-button';
+import { DataSourceToggle } from '@/components/ui/data-source-toggle';
+
+interface DeveloperCardProps {
+  user: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)